extern/guided_diffusion/guided_diffusion/image_datasets.py

```diff
@@ -35,17 +38,30 @@ def load_data(
     :param deterministic: if True, yield results in a deterministic order.
     :param random_crop: if True, randomly crop the images for augmentation.
     :param random_flip: if True, randomly flip the images for augmentation.
+    :param few_shot_count: if not None, the trainer selects few_shot_count samples to train on.
+    :param validation_size: if not 0, also returns a validation data loader with validation_size
+                            samples from the same data_dir, but using disjoint samples.
+                            -- supported only for few shot training! --
     """
     if not data_dir:
         raise ValueError("unspecified data directory")
-    all_files = _list_image_files_recursively(data_dir)
+    
     classes = None
+
+    torch.manual_seed(42)
```

Comment:
is this the right place to seed?

Reply:
https://github.com/talafek96/Diffusion-AD-Project/pull/46